src/common/preferences/preferences.html

```diff
@@ -35,53 +35,51 @@
 				<div class="pane-image"><img src="../images/prefs-general.png" alt=""/></div>
 				<div class="pane-label">General</div>
 			</div>
+			<div id="pane-proxies" class="pane" style="display: none">
+				<div class="pane-image"><img src="../images/prefs-proxies.png" alt=""/></div>
+				<div class="pane-label">Proxies</div>
+			</div>
 			<div id="pane-advanced" class="pane">
 				<div class="pane-image"><img src="../images/prefs-advanced.png" alt=""/></div>
 				<div class="pane-label">Advanced</div>
 			</div>
 		</div>
 		<div id="content-general" class="content">
 			<div class="group">
-				<div class="group-title">Zotero Standalone</div>
+				<div class="group-title">Zotero Status</div>
 				<div class="group-content">
-					<p>Zotero Standalone is currently <span id="general-span-standalone-status"></span>.</p>
+					<p>Zotero is currently <span id="general-span-standalone-status"></span>.</p>
 					<p><input type="button" value="Update Status" id="general-button-update-standalone-status"/></p>
 				</div>
 			</div>
 			<div class="group">
-				<div class="group-title">Save via Server</div>
+				<div class="group-title">Save to Zotero.org</div>
```

Comment:
I don't disagree with you, but there's nothing grosser than capitalized domain names.

Calling it zotero.org also may be a little technical — I've done that a couple places in the client too, but given that people don't use or understand domain names anymore, it may not be the best approach in general. "Save to Web Library" seems a little less clear to me, since the stuff you save to Zotero can end up there too after syncing. Is "Save to Cloud" too buzzwordy? That's sort of the most precise updating of "Save to Server", which I think was clear, if technically inexact.